src/Entity/AdherentCharter/LreCharter.php

```diff
@@ -1,12 +0,0 @@
-<?php
-
-namespace App\Entity\AdherentCharter;
-
-use Doctrine\ORM\Mapping as ORM;
-
-/**
- * @ORM\Entity
- */
-class LreCharter extends AbstractAdherentCharter
```

Comment:
could you add a SQL query into migration for removing all rows for this type of charter from `adherent_charter` table?

same for `LreManagerElectedRepresentativeFilter` and `LreManagerElectedRepresentativeMessage` entities.

Reply:
ok